WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -542,6 +550,9 @@ private extension NoResultsViewController {
     struct NoConnection {
         static let title: String = NSLocalizedString("Unable to load this page right now.", comment: "Title for No results full page screen displayed when there is no connection")
         static let subTitle: String = NSLocalizedString("Check your network connection and try again.", comment: "Subtitle for No results full page screen displayed when there is no connection")
+        static let titlePostList: String = NSLocalizedString("Unable to load posts right now.", comment: "Title for No results full page screen displayedfrom post list when there is no connection")
+        static let subTitlePostList: String = NSLocalizedString("Check your network connection and try again. Or draft a post.", comment: "Subtitle for No results full page screen displayed from post list when there is no connection")
```

Comment:
This message sounds a bit strange due to the "Or draft a post." bit, can I ask you @osullivanchris to share your opinion on it?

Reply:
This _was_ @osullivanchris suggestion. https://github.com/wordpress-mobile/WordPress-iOS/issues/11997#issuecomment-506047116

Reply:
The message was actually written by editorial. If you want me to raise it with them I can? I think its ok! Its slightly awkward, but might just be the nature of what we're trying to say.

Reply:
That's fine - the reason I was asking was to double check as the message looks a bit awkard indeed!  I'm marking this as resolved.